tests/test_exist_client.py

```diff
@@ -75,3 +77,24 @@ def test_url_parsing(url, properties):
     assert client.host == properties[3]
     assert client.port == properties[4]
     assert client.prefix == properties[5]
+
+
+def test_query_with_lengthy_contents(test_client):
+    document = Document("existdb://localhost/exist/db/apps/test-data/dada_manifest.xml")
+    long_paragraph = document.root.full_text * 5  # 30625 characters total length
+    Document(
+        f'<example id="t8"><p>{long_paragraph}</p></example>', existdb_client=test_client
+    ).existdb_store(filename="the_long_dada.xml")
+
+    retrieved_nodes = test_client.xpath(f'//p[contains(., "{long_paragraph}")]')
+    assert len(retrieved_nodes) == 1
+
+
+def test_query_with_cdata_chunk(test_client):
+    Document(
+        f'<example id="t8"><p><![CDATA[What am I doing here]]></p></example>',
+        existdb_client=test_client,
+    ).existdb_store(filename="home_of_cdata.xml")
+
+    retrieved_nodes = test_client.xpath(f'//p[contains(., "What am I doing here")]')
+    assert len(retrieved_nodes) == 1
```

Comment:
due to my previous misconception i'm unsure whether this is needed. b/c i was cautious reg. a `CDATA` fragment inside the query that is wrapped in such within the query paylod.

Reply:
Alles klar, I removed the test